src/strategies_and_params/replicastrategy.jl

```diff
@@ -29,7 +29,7 @@ num_replicas(::ReplicaStrategy{N}) where {N} = N
 Return the names and values of statistics related to `N` replicas consistent with the
 [`ReplicaStrategy`](@ref) `RS`. `names`
 should be a tuple of `Symbol`s or `String`s and `values` should be a tuple of the same
-length. This fuction will be called every [`reporting_interval`](@ref) steps from [`lomc!`](@ref), 
+length. This fuction will be called every [`reporting_interval`](@ref) steps from [`lomc!`](@ref),
```

Comment:
```suggestion
length. This function will be called every [`reporting_interval`](@ref) steps from [`lomc!`](@ref),
```
this was probably my typo originally oops!